README.md

```diff
@@ -23,20 +23,26 @@ Sample Scanner Implementation
 
 Development Requirements
 -------------
-On Fedora or RHEL you can install requirements by issuing
+On RHEL you can install requirements by issuing
 
-    yum install ruby-devel rubygem-ffi rubygem-rake rubygem-bundler openscap
+    yum install ruby-devel rubygem-rake rubygem-bundler openscap
+    gem install ffi # or install rubygem-ffi RPM package from EPEL
+
+On Fedora, command is
+
+    yum install ruby-devel rubygem-rake rubygem-ffi rubygem-bundler openscap
```

Comment:
I would prefer to keep `dnf` command here.

There is no yum in Fedora. It is just a symlink to dnf command. This may feel like a nitpick to you, however, I have experienced with over hundred libraries and opensource programs, usually, when they got something like yum vs dnf wrong on their website, it was a proxy to the quality of the software. Rarely, you will find quality software that says on fedora use Yum.

Next, and that is a nitpick as well, I would like to keep Fedora instructions to the top.

Reply:
Both nitpicks are valid, thanks for pointing them out. Sorry for this PR taking so many iterations.